groups/infrastructure/main.tf

```diff
@@ -0,0 +1,94 @@
+terraform {
+  required_version = "~> 1.3"
+  required_providers {
+    aws = {
+      source  = "hashicorp/aws"
+      version = "~> 4.54.0"
+    }
+    vault = {
+      source  = "hashicorp/vault"
+      version = "~> 3.18.0"
+    }
+  }
+}
+
+provider "aws" {
+  region  = var.aws_region
+}
+
+terraform {
+  backend "s3" {}
+}
+
+module "file_transfer_alb" {
+  source = "git@github.com:companieshouse/terraform-modules//aws/application_load_balancer?ref=1.0.296"
+  count = "${var.file_transfer_create_alb == 1 ? 1 : 0}"
```

Comment:
This conditional uses deprecated interpolation syntax which can be cleaned up.

E.g.:
```
  count = var.file_transfer_create_alb == 1 ? 1 : 0
```

Additionally, it would be benefickal and cleaner to specify `file_transfer_create_alb` as a boolean variable allowing the conditional to be simplified further:
```
  count = var.file_transfer_create_alb ? 1 : 0
```

Reply:
This is fixed now. Commit is pushed.